src/main/java/com/parsing/api/ExtractorLotInformation.java

```diff
@@ -0,0 +1,160 @@
+package com.parsing.api;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.parsing.model.LotResult;
+import com.parsing.model.LotStatus;
+import com.parsing.model.Participant;
+import com.parsing.model.Status;
+import com.parsing.repository.LotResultRepository;
+import com.parsing.repository.ParticipantRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class ExtractorLotInformation {
+
+    ObjectMapper objectMapper = new ObjectMapper();
+    private final String URL_LOT = "https://public.api.openprocurement.org/api/2.5/tenders/";
```

Comment:
what about if the new api version will appear and we need to switch to it? I suppose we'll need to make another deploy instead of changing a property in yaml file

Reply:
ok I will write this to yaml.